bin/nrnmatlab.h

```diff
@@ -42,6 +43,7 @@ void hoc_run(double finitialize_val){
 
 // Run simulation.
 void fadvance(){
+    nrnmpi_stubs();
```

Comment:
Looking at the implementations in neuron itself, this nrnmpi_stubs should already be called (once) before calling ivocmain_session.